connector/pubsub_test.go

```diff
@@ -935,3 +935,27 @@ func TestCannotConnect(t *testing.T) {
 	require.Error(t, err)
 	require.NoError(t, sub.Close())
 }
+
+func TestCredentialsProvider(t *testing.T) {
```

Comment:
This test is not related to Pub Sub i.e. it should not be a part of pubsub_test.go